DECISION-MAKING.md

```diff
@@ -1,143 +1,154 @@
 `Last update: 2018-07`
 
-# Service owners
+# _Mozilla IAM_ Program Strategy
 
-High level service and product owners for Mozilla IAM.
+* @jeffbryner (IT, InfoSec)
+* @hmitsch (Open Innovation, ParSys)
 
-* Capability owner: @jeffbryner
-  * Product owner: @gdestuynder
+# Mozilla IT Operating Model
+
+* IT Capability Owner: @jeffbryner
+  * IT Product Owner: @gdestuynder
     * Auth0 & Auth0 Lock & Auth0 Dashboard: @jdow @akrug @gdestuynder
+       * <Question: Is `Auth0 Lock` the same as `NLX`>
     * Service Integration (CIS): @akrug
     * LDAP: @jdow
     * DuoSecurity: @gdestuynder
-    * Mozillians.org: @hmitsch
-    * WorkDay: HRIS
-
-## Architects
-
-* @gdestuynder (IAM, Security)
-* @mvk-mozilla (IT)
-
-## Program Managers
-
-* @hmitsch
-
-## User Experience (UX)
-
-* @mbranson
+    
+* IT Enterprise Architecture
+   * @mvk-mozilla
+   
+* Outside-IT Capability Ownership
+    * User Experience (UX): @mbranson
+    * Mozillians.org: ParSys
+    * WorkDay (HRIS): People Team
 
-# Modules owners
+# Modules Owners
 
-This document list which team or persons are responsible for which component (module) of the Mozilla IAM stack. This is
-a more detailed model.
+This document list which team or people are responsible for which component (module) of the _IAM Stack_. This is mainly useful for decision making and is inspired by [Mozilla's Module System](https://wiki.mozilla.org/Modules).
 
-This is mainly useful for decision making and is loosely following the [Firefox Module
-system](https://wiki.mozilla.org/Modules).
+_Note:_ We don't yet know what a conflict resolution process would look like if there was a diverging opinion between the Module Owner and the IT Capability Owner for that module.
 
-- **Owner**: A single person which is accountable for the module and normally the final decision maker. In case there is
-  serious disagreement, the Service owner, Product owner, or Capability owner can still be consulted. The Capability
-  owner has final say on all decisions.
-- **Peers**: List of persons which can act on behalf of the Owner, in case the owner is unavailable.
-- **Operator**: A single person or team which is responsible for keeping the system running, generally also the
+Extensions to the Mozilla Module System:
+- **IT Capability Owner**: The person which is capability owner for a particular module.
+- **Operator**: A person or team which is responsible for keeping the system running, generally also the
```

Comment:
disagree with removing the definition of "peer" and "owner" these can be modified but should stay

Reply:
If we use the Mozilla Module System, these terms are defined here: https://www.mozilla.org/en-US/about/governance/policies/module-ownership/

How about a link instead of duplication?

Reply:
i dont know if we want to use their exact doc since we dont exactly have the same process (our definition takes 10s to understand, their takes 10min)